tests/vmi_hook_sidecar_test.go

```diff
@@ -21,9 +21,11 @@ package tests_test
 
 import (
 	"context"
+	_ "embed"
 	"fmt"
 	"time"
 
+	hooksv1alpha1 "kubevirt.io/kubevirt/pkg/hooks/v1alpha1"
```

Comment:
nit: accidental change? probably no need to re-arrange the imports.

Reply:
Neither accidental nor intentional. On my IDE, I have an optimize imports option enabled that does it every time a file is saved. This is the result of that.

Reply:
If it is not intentional, could we revert the change? Previously, the imports were grouped logically. For me, this IDE optimization does not make a lot of sense TBH. In general, I would suggest to disable such settings, since they only add 'noise' to PRs. Or if the imports would really need a reorg, then it could be considered as refactoring and better done in a separate commit with some justification.

Reply:
Sure, I have reverted. Since I have been bitten by it in the past on a different repo in the KubeVirt org, I have it enabled in the IDE. But, in this case, you are 100% right.